config/features.yml

```diff
@@ -2,36 +2,53 @@
 # Add a new feture toggle here to ensure that it is initialized in all environments.
 # Features are defaulted to enabled in development and test environments and disabled in all others.
 # The description should contain any relevant information for an admin who may toggle the feature.
+#
+# The actor_type should be either `user` for featires you want to be "sticky" for a logged in user (default)
+#  or `cookie_id` of you wish to use the Google Analytics id as the unique identifier.
 
 features:
   facility_locator_show_community_cares:
+    actor_type: user
```

Comment:
technically,  `actor_type: user` is optional, but I thought it was more clear this way

Reply:
What happens when `actor_type` is omitted for a given feature config? Is there a validation somewhere / is there a default value?

Reply:
it will [default to the logged in user](https://github.com/department-of-veterans-affairs/vets-api/pull/3868/files#diff-d14889a0f335ac94fca7645d09edb8d6R24).